src/cloud/env/environments.md

```diff
@@ -16,3 +16,12 @@ You can configure application settings, routes, build and deploy actions, and no
 -  [`.magento/services.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_services.html)—defines the services Magento uses by name and version. For example, this file may include versions of MySQL, PHP extensions, Redis, RabbitMQ, and Elasticsearch. You must open a support ticket to push these changes to Pro plan Staging and Production environments.
 
 When you push code changes, the active environment provisions container updates using the YAML configuration files.
+
+## Update Pro Staging and Production environment configuration
+
+For {{site.data.var.ece}} Pro Staging and Production environments, you can update many configuration options in your local development environment and commit the changes to apply them to these environments. However, you must submit a support ticket to update the following configuration options:
+
+-  Install or update services in the `.magento/services.yaml` file.
+-  Change the configuration for the `mounts` and `disk` properties in the `.magento.app.yaml` file.
+
+Some Pro projects might also require a support ticket to update the [route configuration]({{site.baseurl}}/cloud/project/project-conf-files_routes.html) in the `routes.yaml` file, and the [cron configuration]({{site.baseurl}}/cloud/configure/setup-cron-jobs.html) in the `.magento.app.yaml` file.
```

Comment:
Some Pro projects? Do we know the qualifiers? Just curious.

Reply:
Projects where self-service was not enabled. Hm, I not sure that we described somewhere  what it means.

Reply:
It is possible that someone with a Pro project sees this and then wonders if it applies to them. It will be a nice-to-have to mention how to differentiate which Pro projects.
This should not stop the PR, but could be useful information.

Reply:
``` It will be a nice-to-have to mention how to differentiate which Pro projects.```
We don't have a clean way of doing this, other than adding a note to say Contact your CSM, or Submit a support ticket to see whether your project has been enabled for self-service.

Reply:
> Some Pro projects? Do we know the qualifiers? Just curious.

Crons == CRON auto option enabled
routes = NGINX auto option enabled